datafusion/proto/proto/datafusion.proto

```diff
@@ -786,16 +789,21 @@ enum PrimitiveScalarType{
     UTF8 = 11;
     LARGE_UTF8 = 12;
     DATE32 = 13;
-    TIME_MICROSECOND = 14;
-    TIME_NANOSECOND = 15;
+    TIMESTAMP_MICROSECOND = 14;
```

Comment:
I renamed these fields because they are for `Timestamp` not actually `Time` (which are different in Arrow).